### DISCUSSION TITLE
Checksums or PGP signatures

### DISCUSSION THREAD

**Discussion by Clock-Speed** [schemaspy/schemaspy#1250] [Ideas]
Would it be possible to provide checksums for each release to aid with verification. It would be even better if each release was fully signed using PGP/GPG.

**Comment by npetzall** [comment]
We could use the same binaries that we publish to maven or you could fetch from maven.

**Reply by npetzall** [reply]
There is a slight differens in the artifacts so we should look into if the one publish to maven is till available and not cleaned during the build, along with the checksum files.

Maven builds it twice during, and I think we pick the first one, we should pick the second one.

**Reply by Clock-Speed** [reply]
Could you publish your PGP key-id so that we can verify the jar on maven?

**Reply by npetzall** [reply]
0x844EC3471BE008BD
